### ISSUE TITLE
https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/Introduction

### ISSUE THREAD

**Issue by cek333** [mdn/sprints#3592]
# Request type

- [ ] Please close this issue, I accidentally submitted it without adding any details
- [ ] New documentation
- [x] Correction or update

# Details

In the section "Using middleware", one paragraph contains the text "You can add a middleware function to the processing chain with either app.use() or app.add()" ...
I believe it should be app.get() instead of app.add(). I couldn't find any method call add() in the Express documentation.

https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/Introduction

**Comment by chrisdavidmills** [comment]
Another one for @hamishwillee; he'll be working on updating these docs soon.

**Comment by hamishwillee** [comment]
Thanks very much @cek333 .

`app.add()` should have been `app.all()`. However the useful distinction to make was is indeed between `app.use` and the HTTP-verb specific methods: `app.get`/`app.post` etc. I have changed [appropriately](https://wiki.developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/Introduction$compare?locale=en-US&to=1647267&from=1647045).

FYI, I did not include `app.all()` in the discussion. It has subtly different usage semantics than `app.use()` which I think put it beyond scope for this doc.
